ibm/mas_devops/roles/suite_rollback/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+galaxy_info:
+  author: André Marcelino (@andrercm)
+  description: Check an existing IBM Maximo Application Suite v8.11.0 installation is ready to rollback
```

Comment:
We should remove 8.11.0 from this description. As I assume we can rollback 8.12 etc?